classes/services/PKPSubmissionService.inc.php

```diff
@@ -120,6 +121,7 @@ private function _buildGetSubmissionsQueryObject($contextId, $args = array()) {
 			->filterByStageIds($args['stageIds'])
 			->filterByIncomplete($args['isIncomplete'])
 			->filterByOverdue($args['isOverdue'])
+			->filterByInactivity($args['isInactive'])
```

Comment:
Following on from my comment above, this would be called `filterByDaysInactive`.